templates/base.html

```diff
@@ -57,8 +57,7 @@
                     <ul class="vertical menu">
                         <li><a class="nav-color" href="{% url 'home' %}">HOME</a></li>
                         <li><a class="nav-color" href="{% url 'artists' %}">ARTISTS</a></li>
-                        <li><a class="nav-color" href="{% url 'leaderboard' %}">CHARTS</a></li>
-                        <li><a class="nav-color" href="https://blog.investperdiem.com" target="_blank">BLOG</a></li>
```

Comment:
That's a good point. No, I'm fine with leaving the BLOG link off of the nav bar. I might suggest putting BLOG first in the footer though.